hypha/settings/base.py

```diff
@@ -63,9 +63,9 @@
     "SUBMISSIONS_ARCHIVED_ACCESS_STAFF_ADMIN", True
 )
 
-# Possible values are: "public_id" and "title"
+# Possible values are: "application_id", "fund_name" and "title"
 SUBMISSION_TITLE_TEXT_TEMPLATE = env(
-    "SUBMISSION_TITLE_TEMPLATE", default="{title} (#{public_id})"
+    "SUBMISSION_TITLE_TEMPLATE", default="{fund_name} #{application_id} - {title}"
```

Comment:
I find this format more easy to digest, what your thoughts on it? Keep or revert? @frjo @wes-otf